source/deployment-guide/server/kubernetes/deploy-k8s.rst

```diff
@@ -109,7 +109,7 @@ Step 3: Deploy Mattermost
   * ``metadata.name``: The name of your Mattermost deployment in Kubernetes.
   * ``spec.size``: The size of your installation (e.g., "100users", "1000users", etc.).
   * ``spec.ingress.host``: The DNS name for your Mattermost installation.
-  * ``spec.version``: The Mattermost version. See the :doc:`server version archive </product-overview/version-archive>` for available versions.
+  * ``spec.version``: The Mattermost version. See the :doc:`server version archive </product-overview/version-archive>` for available versions. We recommend using a :doc:`supported version </product-overview/release-policy>` of Mattermost in conjunction with the latest version of the Mattermost Operator.
```

Comment:
Perhaps we want to send users [here](product-overview/mattermost-server-releases.html) instead?